ports/bddisasm/portfile.cmake

```
@@ -0,0 +1,29 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO bitdefender/bddisasm
+    REF v1.32.1
+    SHA512 78062081ab38f208c29e1a8cd50daad9203c93ab68cb3e48250fc3b38b7bfdb6a878a995c353f63ac7a6144f305dbdc0f5d60d67558f0403a669197979143de1
+    HEAD_REF master
+)
+
+vcpkg_configure_cmake(

```

```suggestion
vcpkg_cmake_configure(
```
We prefer to use new function instead of the deprecated one for new ports. 
https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_configure_cmake.md

---

I feel like I wasted your time by using the old functions. I did a first draft of this port sometimes in 2020, but haven't made it public until recently. `vcpkg_cmake_configure` wasn't a thing back than (at least I don't remember). I should have checked for updates before opening the PR.

---

For the deprecated functions, this is due to that we have no document to declare we should use new functions for new ports. So this is not wasting time. We need to pick this point up when review PRs. As you can see, there are still many ports using the deprecated functions. Currently, we only suggest to use the new functions for new ports. Other cases are optional.